src/components/Catalog.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+class Catalog extends React.Component {
+  // constructor(props){
+  //   super(props);
+  //   this.state = {
+
+  //   }
+  // }
+  render() {
+    const { phones, onPhoneSelected, addInShopping } = this.props;
+    console.log("CATALOG: ", this.props);
+    return (
+      <ul className="phones">
+  
+        { phones.map(phone => (
+          <li className="thumbnail" key={phone.id} >
+            <a
+              href={'#' + phone.id}
```

Comment:
```suggestion
              href={`#${phone.id}`}
```
Template string is powerful feature